pkg/apis/garden/v1beta1/zz_generated.conversion.go

```diff
@@ -4451,19 +4456,15 @@ func autoConvert_garden_SeedSpec_To_v1beta1_SeedSpec(in *garden.SeedSpec, out *S
 	out.BlockCIDRs = *(*[]v1alpha1.CIDR)(unsafe.Pointer(&in.BlockCIDRs))
 	out.Visible = (*bool)(unsafe.Pointer(in.Visible))
 	out.Protected = (*bool)(unsafe.Pointer(in.Protected))
+	// WARNING: in.Volume requires manual conversion: does not exist in peer-type
```

Comment:
you should address these warnings - even a dummy implementation is better than having these.

Reply:
What do you mean with a dummy implementation? Can you show? I think these warnings just want to remind that no conversion can happen automatically and that you have to manually do it (which I did in `conversions.go`)?!

Reply:
Yes, but you should follow the naming convention - if `conversion-gen` detects that there is such a conversion, it won't bug you and also use it in all `auto_` conversions.

Reply:
Please help me out. What exactly do you suggest to do?

Reply:
Ah I just saw now that this value does not exist in the `v1beta1` type - I thought that there was some way to circumvent the Warning, but I checked the source and there isn't - consider this done.

Reply:
OK, thank you 👍